src/AzureIoTHub.Portal.Infrastructure/DevelopmentConfigHandler.cs

```diff
@@ -24,6 +24,10 @@ internal DevelopmentConfigHandler(IConfiguration config)
 
         public override string IoTHubConnectionString => this.config[IoTHubConnectionStringKey];
 
+        public override string IoTHubEventHubEndpoint => this.config[IoTHubEventHubEndpointKey];
+
+        public override string IoTHubEventHubConsumerGroup => this.config[IoTHubEventHubConsumerGroupKey];
```

Comment:
Could we provide here a default value for the consumer group (if the setting is not provided)?